projects/control-service/projects/helm_charts/pipelines-control-service/templates/deployment.yaml

```diff
@@ -184,6 +184,12 @@ spec:
               value: "{{ .Values.security.authorization.jwtClaimUsername }}"
             - name: DATAJOBS_DEPLOYMENT_JOB_IMAGE_PULL_POLICY
               value: "{{ .Values.dataJob.deployment.jobImagePullPolicy }}"
+            - name: DATAJOBS_DEPLOYMENT_CONFIGURATION_PERSISTENCE_WRITE_TO_K8S
```

Comment:
Do we need two separate configurations? Can't we have one, similar to the "read"? E.g: `DATAJOBS_DEPLOYMENT_CONFIGURATION_PERSISTENCE_WRITE_DATA_SOURCE` with the options "*k8s*", "*db*", "*all*".
I am not sure if it would be ideal, but having so many different configuration variables seems a bit too much, at least in my opinion.

Reply:
My thinking here was extensibility - e.g adding data sources if required in the future would be clearer IMO to provide an additional property with true/false, instead of using a csv list to store toggled properties. I'm not so sure now.